session.go

```diff
@@ -54,16 +54,18 @@ var (
 		"SERIAL      Consistency = 0x0008\n" +
 		"LOCALSERIAL Consistency = 0x0009\n" +
 		"LOCALONE    Consistency = 0x000A")
-	errNoConnection = fmt.Errorf("no working connection")
+	errNoConnection   = fmt.Errorf("no working connection")
+	errNoPreallocated = fmt.Errorf("preallocated handlers size must be greater than 0")
 )
 
 func DefaultSessionConfig(hosts ...string) SessionConfig {
 	return SessionConfig{
 		Hosts: hosts,
 		ConnConfig: transport.ConnConfig{
-			Timeout:            500 * time.Millisecond,
-			TCPNoDelay:         true,
-			DefaultConsistency: LOCALQUORUM,
+			Timeout:              500 * time.Millisecond,
+			TCPNoDelay:           true,
+			DefaultConsistency:   LOCALQUORUM,
+			PreallocatedHandlers: 16000,
```

Comment:
I do not think this needs to be configured.
There is a task to put all constants to ConnConfig so it would be configured later.
Keeping it the same size as request chan size makes sense.

Reply:
Or maybe not, I'd use 1k as a starting point.